src/components/FlowRunTimeline.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="flow-run-timeline-container">
+    <span class="flow-run-timeline-container__beta-badge">
+      Beta
+    </span>
+    <FlowRunTimeline
+      v-if="graphData.length > 0"
+      :graph-data="graphData"
+      :is-running="isRunning"
+    />
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { FlowRunTimeline, TimelineNodeData } from '@prefecthq/graphs'
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useWorkspaceApi } from '@/compositions'
+  import { FlowRun, GraphNode } from '@/models'
+
+  const props = defineProps<{
+    flowRun: FlowRun,
+  }>()
+
+  const api = useWorkspaceApi()
+
+  const graphSubscription = useSubscription(api.flowRuns.getFlowRunsGraph, [props.flowRun.id], { interval: 1000 })
+
+  interface GraphDataTypeGuard extends GraphNode {
+    startTime: Date,
+  }
+
+  const graphData = computed(() => {
+    const items: TimelineNodeData[] = (graphSubscription.response ?? [])
+      .filter((node): node is GraphDataTypeGuard => node.startTime !== null)
+      .map(node => {
+        const nodeData = {
+          id: node.id,
+          label: node.id,
```

Comment:
I might hold off on merging this until the `label` field has been added to nodes, in which case I'll update this PR.

Reply:
https://github.com/PrefectHQ/prefect/pull/7951 - adds names to the `/graph` route, assuming that's what you were looking for